### PR TITLE
[TR] index.md update, translate 07 11 13 14

### PR DESCRIPTION
**Pull request by mekici**
Week 13 added tot he index

### CODE REVIEW

**Review by Atcold — CHANGES_REQUESTED** [review]
I'm confused. Isn't this PR supposed to update `index-tr.md` only?
What are these?

image

**Comment by mekici** [comment]
No, as I said in the previous mail, due to conflicted merge I removed the
fork and commits related to the translations also have gone with the fork.
So I translated the pages 7-11-13-14 again and pushed to the origin. It
contains both the translated pages and updates of index.md.

Murat
_________________________________________________



Dr. Murat EKİCİ, Ph.D.
Usak University
Faculty of Education
Department Of Computer Education and Instructional Technology, Office #329
USAK/TURKEY
T:  +90 276 221 2130 / 2179    F: +90 276 221 2131    Gsm: +90 532 067
88 15

Email: murat.ekici@usak.edu.tr
web: http://akbis.usak.edu.tr/Home/Index/murat.ekici
<http://akbis.usak.edu.tr/Home/Index/murat.ekici>

File: docs/tr/week07/07.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Otokodlayıcıların bazı uygulamalarını ve neden kullanıldığını ele alıyoruz. Ardından, otokodlayıcıların farklı mimarileri, aşırı öğrenme sorunlarından nasıl kaçınılacağı ve kullanmamız gereken kayıp fonksiyonları hakkında konuşacağız. Son olarak, standart bir otokodlayıcı ve gürültü azaltıcı bir otokodlayıcı uygulaması yapacağız.
+
+<!--
+## Practicum
+
+We discussed some applications of Autoencoders and talked about why we want to use them. Then we talked about different architectures of Autoencoders (under or over complete hidden layer), how to avoid overfitting issues and the loss functions we should use. Finally we implemented a standard Autoencoder and a denoising Autoencoder.
+-->
+
+🇹🇷 muratekici
```
Comment (Atcold) [inline docs/tr/week07/07.md:41]:
```suggestion

```

File: docs/tr/week11/11.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+<!--
+## Practicum
+
+This practicum proposed effective policy learning for driving in dense traffic. We trained multiple policies by unrolling a learned model of the real world dynamics by optimizing different cost functions. The idea is to minimize the uncertainty in the model's prediction by introducing a cost term that represents the model's divergence from the states it is trained on.  
+-->
+
+🇹🇷 muratekici
+
```
Comment (Atcold) [inline docs/tr/week11/11.md:43]:
```suggestion

```

File: docs/tr/week13/13.md [hunk 3]
```diff
@@ -1,41 +1,43 @@
...
-we delve into the theory and code of a specific type of GCN known as Residual Gated GCN.
+## Practicum
+
+In this section, we introduce Graph Convolutional Network (GCN) which is one type of architecture that utilizes the structure of data.  Actually, the concept of GCNs is closely related to self-attention. After understanding the general notation, representation and equations of GCN, we delve into the theory and code of a specific type of GCN known as Residual Gated GCN.
 -->
-Bu bölümde, verinin yapısından yararlabilen GCN mimarisini tanıtacağız. GCN yapısının altında yatan konseptin öz-dikkat ile oldukça yakın olduğunu göreceğiz.
-Notasyonumuzu oluşturduktan sonra, GCN'lerin özel bir türü olan Artık Kapılı GCN *(Resıdual Gated GCN)* lerin teorisini ve kodlarını inceleyeceğiz.
+
+🇹🇷 muratekici
+
```
Comment (Atcold) [inline docs/tr/week13/13.md:43]:
```suggestion

```

File: docs/tr/week14/14.md [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
 ## Practicum
-When training highly parametrised models such as deep neural networks there is a risk of overfitting to the training data. 
-This leads to greater generalization error. 
-To help reduce overfitting we can introduce regularization into our training, discouraging certain solutions to decrease the extent to which our models will fit to noise.
+
+When training highly parametrised models such as deep neural networks there is a risk of overfitting to the training data. This leads to greater generalization error. To help reduce overfitting we can introduce regularization into our training, discouraging certain solutions to decrease the extent to which our models will fit to noise.
 -->
+
+
+🇹🇷 muratekici
```
Comment (Atcold) [inline docs/tr/week14/14.md:42]:
```suggestion

```

File: docs/tr/week13/13.md [hunk 5]
```diff
@@ -1,41 +1,43 @@
----
+
+lang: tr
 lang-ref: ch.13
 title: Hafta 13
-lang: tr
-translation-date: 15 Aug 2020
-translator: emirceyani
+translation-date: 12 June 2020
```
Comment (Atcold) [inline docs/tr/week13/13.md:5]:
```suggestion
translation-date: 12 Jun 2020
```

File: docs/tr/week11/11.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+lang: tr
+lang-ref: ch.11
+title: Hafta 11
+translation-date: 12 June 2020
```
Comment (Atcold) [inline docs/tr/week11/11.md:5]:
```suggestion
translation-date: 12 Jun 2020
```

File: docs/tr/week07/07.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+lang: tr
+lang-ref: ch.07
+title: Hafta 7
+translation-date: 12 June 2020
```
Comment (Atcold) [inline docs/tr/week07/07.md:5]:
```suggestion
translation-date: 12 Jun 2020
```

File: docs/tr/week14/14.md [hunk 8]
```diff
@@ -1,42 +1,42 @@
----
-lang-ref: ch.14
+
 lang: tr
-title: Week 14
-translation-date: 22 Aug 2020
-translator: emirceyani
+lang-ref: ch.14
+title: Hafta 14
+translation-date: 12 June 2020
```
Comment (Atcold) [inline docs/tr/week14/14.md:5]:
```suggestion
translation-date: 12 Jun 2020
```

**Review by Atcold — CHANGES_REQUESTED** [review]
I see, thanks.

You need to update `docs/_config.yml` as well.
The authorship is done by my script, using the `translator:` field. Please, remove the signatures.
Update the name of this PR to reflect the files you're adding.

**Review by Atcold — APPROVED** [review]
Great! Thanks!

**Pull request merged**